COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -639,15 +525,11 @@ In the example above, multiple extensions are installed. The structure of comman
 Don’t have the CLI installed yet? You can also check out a copy of the web help for the core Zowe CLI and Zowe plug-ins [here](https://docs.zowe.org/stable/web_help/index.html).
 
 ### Zowe Profiles
-Zowe client technologies like Zowe CLI and the Zowe Explorer VS Code Extension store connection information in files commonly known as profiles. This provides convenience as after profiles for services are created, users do not have to constantly provide this information. For the secure storage of credentials, there is the Secure Credential Store plug-in which is discussed more in a later section titled "Installation of Zowe CLI and plug-ins". The Secure Credential Store provides a means to store creds in the operating system’s secure credential vault.
-
-When creating profiles you can also specify the `prompt*` keyword to be prompted for your username and password so they will be masked on the command line. Figure 3 shows a sample command to create a zosmf profile. This will eliminate the need to provide these details on future commands.
-
-![](Images/zowe/zowe-cli-zosmf-profile-creation-command.png)
 
-*Figure 3. Zowe CLI z/OSMF Profile Creation Command*
```

Comment:
Since `Figure 3` was removed, it seems that the numbering for subsequent images are wrong. (i.e., after `Figure 2. Zowe CLI Web Help`, it goes to `Figure 4. Zowe CLI zos-files actions`)

Reply:
Fixed issue!